Document/0x06d-Testing-Data-Storage.md

```diff
@@ -85,15 +56,37 @@ The `NSUserDefaults` class provides a programmatic interface for interacting wit
 
 #### Remediation
 
--- TODO [Add content on remediation on "Testing for Sensitive Data Disclosure in Local Storage"]
+If sensitive information (credentials, keys, PII, etc.) is needed locally on the device, several best practices are offered by iOS that should be used to store data securely instead of reinventing the wheel or leave it unencrypted on the device.
+
+The following is a list of best practice used for secure storage of certificates and keys and sensitve data in general:
+* For small amounts of sensitive data such as credentials or keys use the [Keychain Services](https://developer.apple.com/reference/security/1658642-keychain_services?language=objc) to securely store it locally on the device. Keychain data is protected using a class structure similar to the one used in file Data Protection. These classes have behaviors equivalent to file Data Protection classes, but use distinct keys and are part of APIs that are named differently. The the default behaviour is `kSecAttrAccessibleWhenUnlocked`. For more information have a look at the available modes [Keychain Item Accessibility](https://developer.apple.com/reference/security/1658642-keychain_services/1663541-keychain_item_accessibility_cons).
+* Cryptographic functions that have been self implemented to encrypt or decrypt local files should be avoided.  
+* Avoid insecure storage functions for sensitive information such as credentials and keys as illustrated in chapter OMTG-DATAST-001-2.   
+
 
 #### References
 
+* [Keychain Services Programming Guide](https://developer.apple.com/library/content/documentation/Security/Conceptual/keychainServConcepts/iPhoneTasks/iPhoneTasks.html)
```

Comment:
Please update references according to new structure https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md#external-references

Can you assign the ticket to yourself https://github.com/OWASP/owasp-mstg/issues/334